configuration/config2/EnvironmentNode.cpp

```diff
@@ -19,36 +19,36 @@ limitations under the License.
 
 void EnvironmentNode::addChild(std::shared_ptr<EnvironmentNode> pNode)
 {
-	m_children.insert(std::make_pair(pNode->getName(), pNode));
+    m_children.insert(std::make_pair(pNode->getName(), pNode));
 }
 
 
 void EnvironmentNode::addAttribute(const std::string &name, std::shared_ptr<EnvValue> pValue)
 {
-	auto retValue = m_attributes.insert(std::make_pair(name, pValue));
-	// todo: add check to make sure no duplicate attributes, use retValue to see if value was inserted or not
+    auto retValue = m_attributes.insert(std::make_pair(name, pValue));
+    // todo: add check to make sure no duplicate attributes, use retValue to see if value was inserted or not
 }
 
 
 std::vector<std::shared_ptr<EnvironmentNode>> EnvironmentNode::getChildren(const std::string &name) const
 {
-	std::vector<std::shared_ptr<EnvironmentNode>> childNodes;
-	if (name == "") {
-		for (auto nodeIt = m_children.begin(); nodeIt != m_children.end(); ++nodeIt)
-		{
-			childNodes.push_back(nodeIt->second);
-		}
-	}
-	else
-	{
-		auto rangeIt = m_children.equal_range(name);
-		for (auto it = rangeIt.first; it != rangeIt.second; ++it)
-		{
-			childNodes.push_back(it->second);
-		}
-	}
+    std::vector<std::shared_ptr<EnvironmentNode>> childNodes;
+    if (name == "") {
```

Comment:
can we condense the if and else loop into one, by setting the rangeIt appropriately?